Tools/CISupport/ews-build/steps.py

```diff
@@ -3799,6 +3811,14 @@ def evaluateCommand(self, cmd):
             message = 'Passed JSC tests'
             self.descriptionDone = message
             self.build.results = SUCCESS
+        elif (self.preexisting_failures_in_results_db and not self.stressTestFailures_filtered and not self.binaryFailures_filtered):
+            # This means all the tests which failed in this run were also failing or flaky in results database
+            message = f"Ignored pre-existing failure: {', '.join(self.preexisting_failures_in_results_db)}"
+            self.descriptionDone = message
+            self.build.results = SUCCESS
+            self.setProperty('build_summary', message)
+            steps_to_add += [ArchiveTestResults(), UploadTestResults(), ExtractTestResults()]
+            self.finished(WARNINGS)
```

Comment:
finished() is no longer present while running with latest buildbot. Let's replace it with `return WARNINGS`.